auction9-backend/handler.js

```diff
@@ -73,9 +71,9 @@ export const getActiveAuctions = async (event, context) => {
 export const postAuction = async (event, context) => {
   try {
     let reqBody = JSON.parse(event.body);
-    var startDate = new Date(reqBody.date_from).valueOf();
-    var endDate = new Date(reqBody.date_to).valueOf();
-    let status = startDate > Date.now() ? 'INACTIVE' : 'ACTIVE';
+    let startDate = new Date(reqBody.date_from).valueOf();
+    let endDate = new Date(reqBody.date_to).valueOf();
+    let status = startDate > Date.now() ? statuses.inactive : statuses.active;
```

Comment:
On auction create, is it possible to create auction which start time has passed?

Reply:
Dragana made this function, i just changed the way we use statuses. But it's not possible to create an auction which start time has passed, by default it's inactive. I added condition for that now